moped-editor/src/layouts/DashboardLayout/NavBar/useNavigationSearchQuery.js

```diff
@@ -0,0 +1,47 @@
+import { useMemo } from "react";
+import { gql } from "apollo-boost";
+import { NAVIGATION_SEARCH_QUERY_CONFIG } from "./NavigationSearchQueryConf";
+import { getSearchValue } from "src/utils/gridTableHelpers";
+
+export const useNavigationSearch = ({ searchTerm }) => {
+  const columnsToReturn = Object.keys(NAVIGATION_SEARCH_QUERY_CONFIG.columns);
+  const query = useMemo(() => {
+    const searchOperatorsOrValues = searchTerm
+      ? columnsToReturn.map((column) => {
+          const { operator, quoted, envelope } =
+            NAVIGATION_SEARCH_QUERY_CONFIG.columns[column].search;
+          const searchValue = getSearchValue(
+            NAVIGATION_SEARCH_QUERY_CONFIG,
+            column,
+            searchTerm
+          );
+          const graphqlSearchValue = quoted
+            ? `"${envelope.replace("{VALUE}", searchValue)}"`
+            : searchValue;
+
+          return `{ ${column}: {${operator}: ${graphqlSearchValue}} }`;
+        })
+      : [];
+    const searchWhereString = searchOperatorsOrValues.join(", ");
+    const queryString = `query NavigationProjectsList {
+      project_list_view (
+          limit: 10
+          order_by: { updated_at: desc },
```

Comment:
wondering if i should keep these in the config, so they are easier to find?

Reply:
also I am limiting to 10 arbitrarily, we used to have a limit of 250, but the navigation search results will never show anything over 5

Reply:
love the update because there is no way we need 250 results. 🙏 I think that having these set in the query config could be nice to align with how there are set in the existing `PROJECT_LIST_VIEW_QUERY_CONFIG` default limit and order.

Reply:
i moved it to the config 🚚